operator/duties/controller.go

```diff
@@ -241,6 +244,55 @@ func (dc *dutyController) handleSlot(logger *zap.Logger, slot phase0.Slot) {
 		go dc.onDuty(logger, &duties[i])
 	}
 
+	dc.handleSyncCommittee(logger, slot, syncPeriod)
+	dc.handleValidatorRegistration(logger, slot)
+}
+
+func (dc *dutyController) handleValidatorRegistration(logger *zap.Logger, slot phase0.Slot) {
```

Comment:
does validatorRegistration count as a duty in eth? otherwise why is this code here?

Reply:
also this is added as part of MEV, should we do validatorregistration only when MEV is enabled?

Reply:
> does validatorRegistration count as a duty in eth? otherwise why is this code here?
It's not a direct duty, but kind of an indirect one: Validators need to do that regularly to use MEV. Operators need to reach a consensus of partial signatures to build a full signature, so its designed like a duty and fits here well.

> also this is added as part of MEV, should we do validatorregistration only when MEV is enabled?
I think yes